packages/gtk3.rb

```diff
@@ -0,0 +1,36 @@
+require 'package'
+
+class Gtk3 < Package
+  description 'Gtk3 is a cross-platform widget toolkit for creating graphical user interfaces.'
+  homepage 'https://developer.gnome.org/gtk3/3.0/'
+  version '3.26'
+  source_url 'http://ftp.gnome.org/pub/gnome/sources/gtk+/3.22/gtk+-3.22.26.tar.xz'
+  source_sha256 '61eef0d320e541976e2dfe445729f12b5ade53050ee9de6184235cb60cd4b967'
+
+  depends_on 'glib'
+  depends_on 'gobject_introspection'
+  depends_on 'gdk_pixbuf'
+  depends_on 'cairo'
+  depends_on 'pango'
+  depends_on 'libepoxy'
+  depends_on 'graphene'
+  depends_on 'atk'
+  depends_on 'libxkbcommon'
+  depends_on 'meson'
```

Comment:
Since meson already includes python27 and libx11 as dependencies, they don't need to be included here.